src/NumericalAlgorithms/DiscontinuousGalerkin/MetricIdentityJacobian.hpp

```diff
@@ -175,12 +175,77 @@ namespace dg {
  * The subtraction technique is most commonly used in finite difference codes.
  */
 template <size_t Dim>
-void metric_identity_jacobian(
+void metric_identity_det_jac_times_inv_jac(
     gsl::not_null<
         InverseJacobian<DataVector, Dim, Frame::Logical, Frame::Inertial>*>
         det_jac_times_inverse_jacobian,
     const Mesh<Dim>& mesh,
     const tnsr::I<DataVector, Dim, Frame::Inertial>& inertial_coords,
     const Jacobian<DataVector, Dim, Frame::Logical, Frame::Inertial>&
         jacobian) noexcept;
+
+/*!
+ * \ingroup DiscontinuousGalerkinGroup
+ * \brief Compute the Jacobian, inverse Jacobian, and determinant of the
+ * Jacobian so that they satisfy the metric identities.
+ *
+ * Uses `dg::metric_identity_jacobian()` to compute the determinant of the
+ * Jacobian times the inverse Jacobian. Taking the determinant in \f$d\f$
+ * dimensions:
```

Comment:
I suggest expanding a bit for clarity: "By taking the determinant of this product, we can isolate $J$, the determinant of the Jacobian. In $d$ dimensions, we have:" or similar.